src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpoint.java

```diff
@@ -528,5 +530,77 @@ private CollectionExerciseDTO addCaseTypesandSurveyId(CollectionExercise collect
     return ResponseEntity.ok(result);
   }
 
+  /**
+   * PUT request to update a collection event date
+   * @param id Collection exercise Id
+   * @param tag collection exercise event tag
+   * @throws CTPException on resource not found
+   * @return 200 if all is ok, 400 for bad request, 409 for conflict
+   */
+  @RequestMapping(value = "/{id}/events/{tag}", method = RequestMethod.PUT, consumes = "text/plain")
+  public ResponseEntity<?> updateEventDate(
+          @PathVariable("id") final UUID id,
+          @PathVariable("tag") final String tag,
+          final @RequestBody String date)
+          throws CTPException {
+
+    Date finalDate = null;
+
+    log.info("Adding collection exercise {}, setting date to {}", id, date);
+
+      try
+        {
+          LocalDateTime parseDate = LocalDateTime.parse(date, DateTimeFormatter.ofPattern("uuuu-MM-dd'T'HH:mm:ss.SSSX", Locale.ROOT));
```

Comment:
looks like the pattern is an iso 8601 datetime. Would https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME work?

Reply:
Didn't work for me - only worked if there was a ':' in the middle of the timezone